File: TransCoder_tokenized_test_set_functions/python/SORTING_USING_TRIVIAL_HASH_FUNCTION.py
def sortUsingHash ( a, n ) :
    Max = max ( a )
    Hash = [ 0 ] * ( Max + 1 )
    for i in range ( 0, n ) :
        Hash [ a [ i ] ] += 1
    for i in range ( 0, Max + 1 ) :
        if Hash [ i ] != 0 :
            for j in range ( 0, Hash [ i ] ) :
                print ( i, end = " " )
    

File: TransCoder_tokenized_test_set_functions/java/SORTING_USING_TRIVIAL_HASH_FUNCTION.java
static void sortUsingHash ( int a [ ], int n ) {
  int max = Arrays . stream ( a ) . max ( ) . getAsInt ( );
  int hash [ ] = new int [ max + 1 ];
  for ( int i = 0;
  i < n;
  i ++ ) hash [ a [ i ] ] += 1;
  for ( int i = 0;
  i <= max;
  i ++ ) {
    if ( hash [ i ] != 0 ) {
      for ( int j = 0;
      j < hash [ i ];
      j ++ ) {
        System . out . print ( i + " " );
      }
    }
  }
}